<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>

<head>
  <meta name=viewport content=“width=800”>
  <meta name="generator" content="HTML Tidy for Linux/x86 (vers 11 February 2007), see www.w3.org">
  <style type="text/css">
    /* Color scheme stolen from Sergey Karayev */

    a {
      color: #1772d0;
      text-decoration: none;
    }

    a:focus,
    a:hover {
      color: #f09228;
      text-decoration: none;
    }

    body,
    td,
    th,
    tr,
    p,
    a {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px
    }

    strong {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px;
    }

    heading {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 22px;
    }

    papertitle {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px;
      font-weight: 700
    }

    name {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 32px;
    }

    .one {
      width: 160px;
      height: 160px;
      position: relative;
    }

    .two {
      width: 160px;
      height: 160px;
      position: absolute;
      transition: opacity .2s ease-in-out;
      -moz-transition: opacity .2s ease-in-out;
      -webkit-transition: opacity .2s ease-in-out;
    }

    .fade {
      transition: opacity .2s ease-in-out;
      -moz-transition: opacity .2s ease-in-out;
      -webkit-transition: opacity .2s ease-in-out;
    }

    span.highlight {
      background-color: #ffffd0;
    }
  </style>
  <link rel="icon" type="image/png" href="icon.png">
  <title>Shichen Liu</title>
  <meta http-equiv="Content-Type" content="text/html; charset=us-ascii">
  <link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
</head>

<body>
  <table width="800" border="0" align="center" cellspacing="0" cellpadding="0">
    <tr>
      <td>
        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="67%" valign="middle">
              <p align="center">
                <name>Shichen Liu 刘世琛</name>
              </p>
              <p>
              I am a second year PhD student in <a href="https://www.bbe.caltech.edu/academics/bioengineering">Bioengineering</a> at <a href="https://www.caltech.edu/">Caltech</a>, where on study collective behavior of active biomolecules. My PhD advisor is <a href="https://thomsonlab.caltech.edu/team/">Matt Thomson</a>
              </p>

              <p>
              From 2019 to 2021, I worked in the Department of <a href="https://ds.dfci.harvard.edu/">Data Science </a>at <a href="https://www.dana-farber.org/">Dana-Farber Cancer Institute </a> and <a href="https://sysbio.med.harvard.edu/">Systems Biology</a> at <a href="https://hms.harvard.edu/">Harvard Medical School</a> with <a href="https://sysbio.med.harvard.edu/sahand-hormoz">Sahand Hormoz</a>.
              I primarily worked developing single-cell and DNA technologies to investigate cell differentiation and cancer development. I did my undergraduate studies in <a href="https://engineering.case.edu/mechanical-and-aerospace-engineering">Mechanical Engineering and Aerospace Engineering</a> at <a href="https://case.edu/">CWRU</a>, where I researched on sickle cell disease in <a href="https://case.edu/issacs/facultyfaculty-associates/umut-gurkan">Umut Gurkan</a>'s lab.
              </p>

              <p align=center>
                sliu7[at]caltech.edu</a> &nbsp/&nbsp
                <a href="https://scholar.google.com/citations?user=nTj6hGcAAAAJ&hl=en">Google Scholar</a> &nbsp/&nbsp
                <a href="https://twitter.com/shichen_liu97">Twitter</a> &nbsp/&nbsp
                <a href="https://gitlab.com/shichen.liu97">GitLab</a> &nbsp/&nbsp
                <a href="https://github.com/shichenliu97">GitHub
              </p>
            </td>
            <td width="33%">
              <img src="headshot.jpg" width='100%'>
            </td>
          </tr>
        </table>

        <hr>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="100%" valign="middle">
	          <p>

              <p>
                <strong style="color:red;">News:</strong> (Dec 2022) I'll be presenting my recent work on programmed transitions between micro- and millimeter order through microtubule bundling in cytoskeletal networks at Winter qBio 2023. Please come and say hi!
              </p>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="100%" valign="middle">
              <heading>Research and Selected Publications</heading>
              <br>
              <br>
              <p>My research interests are in the intersection of synthetic biology and physics. Both living and non-living
                <a href="https://thomsonlab.caltech.edu/active-matter/">active matters</a>
                create non-equilibrium processes in which many individual parts consumes energy and collectively generate motion and force.
                Active matter exhibit many interesting properties, and I am most interested in the phase transition between local and global/collective
                behavior. We use components from the cytoskeleton, motor proteins and microtubules, as a model system to decode non-equilibrium dyanmics
                of active matters. I use theortical and experimental tools to investigate how the flow of time and the "greed" of nature evolve or alter
                active materials. </p>
                I aim to: (1) understand and control the transition between local and global behavior, (2) bridge our unsderstanding of
                biomolecules from cytoskeleton and mechanism of neurodegenerative diseases, and (3) design programmable active material using a number of novel biomolecules.</p>

                Aside from active matters, I'm also interested in developmental biology and biomechanics. I previously worked on developing single-cell technologies
                to measure cell states in diverse biological systems (mainly on bacterial and mammalian cells). I also worked on building organoid systems based on
                understanding obtained from our high resolution cell state data.



              <p>For an up-to-date and full publication list, please see my <a href="https://scholar.google.com/citations?user=nTj6hGcAAAAJ&hl=en">Google Scholar</a> page.</p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="85%" valign="left">
              <papertitle><em>Active Matters</em>: Decipher and Program the Dynamical Behaviours of Self-Organizing Subcellular Structures\</papertitle>
              <hr>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width:35%;vertical-align:middle">
              <img src='my_arxiv.gif' width=100%>
            </td>
            <td style="width:65%;vertical-align:middle">
              <p>
                  <papertitle>Programmed transitions between micro- and millimeter order through microtubule bundling in cytoskeletal networks </papertitle>
                </a>
                <br>
                <strong>[Manuscript in progress]</strong>
                <p>Active molecule induced movements are responsible for locomotion, reconfiguration and replication of living systems.
                  Experimental active-matter systems consist of motor proteins and microtubules are capable of multiscale
                  self-organization ranging from contractile and extensile phases with millimeter-scale behaviors to local aster
                  phases that only span a few micrometers. Here we show that the bundling effect of microtubules drives the phase
                  transition between global and local self-organization or our kinesin-microtubule experimental system.
                </p>
            </td>
          </tr>

          <tr>
            <td style="width:35%;vertical-align:middle">
              <img src='fan_arxiv.gif' width='100%'>
            </td>
            <td style="width:65%;vertical-align:middle">
              <p>
                <a href="https://arxiv.org/abs/2208.12839">
                  <papertitle>Dynamic Flow Control Through Active Matter Programming Language</papertitle>
                </a>
                <br>
                Fan Yang, <strong>Shichen Liu</strong>, Heun Jin Lee, Matt Thomson
                <br>
                <em>arXiv</em>, 2022
                <br>
                <p></p>
                <p>Active fluids with such living matter can be chaotic without a control mechanism, limiting their
                  utility for bioengineering applications. Recently-developed light-activatable motors, which only
                  crosslink the microtubules under illumination, have been previously applied to sculpt persistent
                  fluid flows. Combining theoretical predictions and experimental results, we find that the linear
                  superimposition can be achieved above a critical distance. Therefore, different flows can be realized
                  simultaneously within a single channel through the spatio-temporal composition and manipulation of
                  light patterns, making multitasking possible in a single channel. Our theoretical framework may be
                  extended for other emergent structures of cytoskeleton, and our platform has potentials for building
                  optically programmable flow-control devices.</p>
            </td>
          </tr>


        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="85%" valign="left">
              <papertitle>Measuring Molecular States of Single Cells in Diverse Biological Systems</papertitle>
              <hr>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width:30%;vertical-align:middle">
              <img src='star_protocol.jpg' width='100%'>
            </td>
            <td style="width:70%;vertical-align:middle">
              <p>
                <a href="https://arxiv.org/abs/2106.06098">
                  <papertitle>Integrating Readout of Somatic Mutations in Individual Cells with Single-Cell Transcriptional
                    Profiling</papertitle>
                </a>
                <br>
                <strong>Shichen Liu</strong>, Maximilian Nguyen, Sahand Hormoz
                <br>
                <em>Star Protocols</em>, 2021
                <br>
                [<a href="https://gitlab.com/hormozlab/scamplicon-library-analysis">code</a>]
                <br>
                <p></p>
                <p>In many biological applications, the readout of somatic mutations in individual cells is essential.
                  For example, it can be used to mark individual cancer cells or identify progenies of a stem cell. Here,
                  we present a protocol to perform single-cell RNA-seq and single-cell amplicon-seq using 10X Chromium
                  technology. Our protocol demonstrates how to (1) isolate CD34+ progenitor cells from human bone marrow
                  aspirate, (2) prepare single-cell amplicon libraries, and (3) analyze the libraries to assign somatic
                  mutations to individual cells.
                </p>
            </td>
          </tr>

          <tr>
            <td style="width:30%;vertical-align:middle">
              <img src='csc.jpg' width='100%'>
            </td>
            <td style="width:70%;vertical-align:middle">
              <p>
                <a href="https://www.sciencedirect.com/science/article/pii/S1934590921000515">
                  <papertitle>Reconstructing the Lineage Histories and Differentiation Trajectories of Individual
                    Cancer Cells in Myeloproliferative Neoplasms</papertitle>
                </a>
                <br>




                Debra Van Egeren, Javier Escabi, Maximilian Nguyen, <strong>Shichen Liu</strong>, Christopher R. Reilly,
                Sachin Patel, Baransel Kamaz, Maria Kalyva, Daniel J. DeAngelo, Ilene Galinsky, Martha Wadleigh, Eric S. Winer,
                Marlise R. Luskin, Richard M. Stone, Jacqueline S. Garcia, Gabriela S. Hobbs, Fernando D. Camargo, Franziska Michor,
                Ann Mullally, Isidro Cortes-Ciriano, Sahand Hormoz

                <br>
                <em>Cell Stem Cell</em>, 2021
                <br>
                [<a href="https://gitlab.com/hormozlab/mpn-scrnaseq-analysis">code</a>]
                <br>
                <p></p>
                <p>Blood cancers known as myeloproliferative neoplasms (MPNs) are thought to originate when a driver mutation is
                  acquired by a hematopoietic stem cell (HSC). Here we quantified the effect of the JAK2-V617F
                  mutation on the self-renewal and differentiation dynamics of HSCs in treatment-naive individuals with MPNs
                  and reconstructed lineage histories of individual HSCs using somatic mutation patterns. We found that JAK2-V617F
                  mutations occurred in a single HSC several decades before MPN diagnosis and found that mutant HSCs have a selective
                  advantage. These results highlight the potential of harnessing somatic mutations to
                  reconstruct cancer lineages.</p>
            </td>
          </tr>

          <tr>
            <td style="width:30%;vertical-align:middle">
              <img src='bacterial.png' width='100%'>
            </td>
            <td style="width:70%;vertical-align:middle">
              <p>
                <a href="https://www.biorxiv.org/content/10.1101/2021.03.10.434868v2.abstract">
                  <papertitle>Droplet-Based Single Cell RNA Sequencing of Bacteria Identifies Known and Previously Unseen Cellular
                    States</papertitle>
                </a>
                <br>
                Ryan McNulty, Duluxan Sritharan, <strong>Shichen Liu</strong>, Sahand Hormoz, Adam Z. Rosenthal
                <br>
                <em>bioRxiv</em>, 2021
                <br>
                [<a href="https://gitlab.com/hormozlab/bacteria_scrnaseq">code</a>]

                <br>
                <p></p>
                <p>Here, we develop a method that uses DNA probes and leverages an existing commercial
                  microfluidic platform (10X Chromium) to conduct bacterial single cell RNA sequencing. We
                  sequenced the transcriptome of over 15,000 individual bacterial cells, detecting on average 365
                  transcripts mapping to 265 genes per cell in B. subtilis and 329 transcripts mapping to 149 genes
                  per cell in E. coli. Our findings correctly identify known cell states and uncover previously
                  unreported cell states.
            </td>
          </tr>

          <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
            <tr>
              <td width="100%" valign="middle">
  	          <p>



                <hr>

                <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
                  <tr>
                    <td width="100%" valign="middle">



              <heading>Other Stuff</heading>
              <p>
                I review for Star Protocols, an open-access peer-reviewed protocol journal from Cell Press.</p>
                I have a serious commitment towards cultivating future researchers who are interested in the intersection of
                biology and physics. I have mentored a number of undergraduate students and research technicians. Feel free to reach
                out if you are interested in the research we do here, and I would love to chat about opportunities.</p>
                I am passionate about automotive racing, occansionally you can find me racing at Willow Springs International Raceway
                (both physically and in sim racing).</p>
                I also enjoying bloudering/climbing, making random stuff, Amateur radio (KE8GFG), and film photography.

         </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
            <td width="50%"><img src="bos1.jpg" alt="prl" width="360" height="239"></td>
            <td width="50%"><img src="bos2.jpg" alt="prl" width="360" height="239"></td>
        </table>
        <table width="100%" align="center" border="0" cellpadding="10">
            <td width="100%" align="center">November, Boston MA</td>
        </table>
        <table width="100%" align="center" border="0" cellpadding="20">
            <td width="50%"><img src="trt1.jpg" alt="prl" width="360" height="239"></td>
            <td width="50%"><img src="trt2.jpg" alt="prl" width="360" height="239"></td>
        </table>
        <table width="100%" align="center" border="0" cellpadding="10">
            <td width="50%" align="center">Gull bully, Toronto</td>
            <td width="50%" align="center">Skyline, Toronto</td>
        </table>
        <table width="100%" align="center" border="0" cellpadding="20">
            <td width="50%"><img src="SH1.jpg" alt="prl" width="360" height="360"></td>
            <td width="50%"><img src="rose2.jpg" alt="prl" width="360" height="360"></td>
        </table>
        <table width="100%" align="center" border="0" cellpadding="10">
            <td width="50%" align="center">Experimenting with slide films, Shanghai</td>
            <td width="50%" align="center">Metal rose the 3rd</td>
        </table>
        <table width="100%" align="center" border="0" cellpadding="20">
            <td width="50%"><img src="ice1.png" alt="prl" width="360" height="360"></td>
            <td width="50%"><img src="ice2.png" alt="prl" width="360" height="360"></td>
        </table>
        <table width="100%" align="center" border="0" cellpadding="10">
            <td width="100%" align="center">Ice climbing at Hyalite Fall, Montana</td>
        </table>

        <hr>

        <table width="80%" align="left" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td>
              <br>
              <p align="left">
                <font size="2">
                  Design and source code from  <a href="https://jonbarron.info/">Jon Barron's website</a>.
                  </font>
              </p>
            </td>
          </tr>
        </table>
        <table width="20%" align="right" border="0" cellspacing="0" cellpadding="20">



        <script type="text/javascript" id="clustrmaps" src="//clustrmaps.com/map_v2.js?d=fsGvgOYKRTjQ3-pqjVm0nDcqEmc_kDj9HrTqBvEfVI0&cl=ffffff&w=a"></script>
        </td>
    </tr>
  </table>
</body>

</html>
